Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,32 @@
+/*In this kata, you will need to implement a function that sums the numerical
+ elements of an array
+
+
+E.g. sum([10, 20, 30]), target output: 60
+E.g. sum(['hey', 10, 'hi', 60, 10]), target output: 80 (ignore any non-numerical elements)
+*/
+let a = [3, 5, 7, 9, 10];
+let result = sum(a);
+console.log(result);
+
+
 function sum(elements) {
+    let total = 0; // Start with 0
+
+    // Loop through each element in the array
+    for (let i = 0; i < elements.length; i++) {
+        // Check if the element is a number
+        if (typeof elements[i] === "number") {
+            total = total + elements[i]; // Add the number to total
+        }
+    }
+
+    return total; // Return the final total sum
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```